win32/zlib-ng.def.in

```diff
@@ -0,0 +1,60 @@
+; zlib-ng data compression library
+EXPORTS
+; basic functions
+    @ZLIB_SYMBOL_PREFIX@zlibng_version
+    @ZLIB_SYMBOL_PREFIX@zng_deflate
```

Comment:
Why are these `@ZLIB_SYMBOL_PREFIX@zng` and not just `@ZLIB_SYMBOL_PREFIX@`?

Reply:
That is completely arbitrary, and I did it because it was easier to copy-paste without having to pay attention which functions have `zng_` prefix.